Networking Sprint Challenge - PokeDex/Networking Sprint Challenge - PokeDex/View Controllers/SearchViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SearchViewController.swift
+//  Networking Sprint Challenge - PokeDex
+//
+//  Created by Vijay Das on 2/1/19.
+//  Copyright © 2019 Vijay Das. All rights reserved.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController, UISearchBarDelegate {
+    
+    // OUTLETS
+    
+    @IBOutlet weak var resultImage: UIImageView!
+    
+    @IBOutlet weak var resultNameLabel: UILabel!
+    @IBOutlet weak var resultIDLabel: UILabel!
+    @IBOutlet weak var resultTypeLabel: UILabel!
+    @IBOutlet weak var resultAbilitiesLabel: UILabel!
+    
+    var pokemonController: PokemonController?
+    
+    var result: Pokemon? {
+        didSet {
+            updateViews()
+        }
+    }
+
+
+
+// updateViews method needs to be added
+    
+    func updateViews() {
+        guard let result = result else { return}
+        resultNameLabel.text = result.name.capitalized
+        resultIDLabel.text = "\(result.id)"
+        
+        
+        let ability = result.abilities.map { $0.subAbility.name}
+        let type = result.type.map { $0.type}
+        
+        resultAbilitiesLabel.text = ability.first
+        resultTypeLabel.text = 
```

Comment:
You're not setting the label to anything and it's causing an error. You should set it to the constant "type" that you created above it.